canvasapi/assignment.py

```diff
@@ -160,6 +160,28 @@ def get_peer_reviews(self, **kwargs):
             _kwargs=combine_kwargs(**kwargs),
         )
 
+    def get_provisional_grades_status(self, **kwargs):
+        """
+        Tell whether the student's submission needs one or more provisional grades.
+
+        :calls: `GET /api/v1/courses:course_id/provisional_grades/status \
+        <https://canvas.instructure.com/doc/api/all_resources.html#method.provisional_grades.status>`_
+        :param user: User that will be checked for provisional grades
+        :type user: :class:`canvasapi.user.User`
```

Comment:
These docs specify a `user` parameter, but one doesn't appear in the method signature. According to Canvas' docs, there is a `student_id` parameter.

They don't list the `student_id` as required (although it probably actually is). Recommend adding and handling `student_id` to args and updating the docstring. (also, should accept `User` object or `int`. Use `obj_or_id` for this)

Alternatively, remove the docstring and let the user pass it as a kwarg.

Reply:
Is it fine if I just use obj_or_int to get the student_id opposed to passing it in?